src/main/java/com/google/sps/data/SessionContext.java

```diff
@@ -3,14 +3,16 @@
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.sps.data.NonPersistentUserRepository;
```

Comment:
The NonPersistentUserRepository import can be removed (L5).

Reply:
Done